projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/logins-blocked-status.tsx

```diff
@@ -54,9 +55,16 @@ function BlockedStatus( { status }: { status: 'active' | 'inactive' | 'off' } )
 	const tooltipContent = useProtectTooltipCopy();
 	const { blockedLoginsTooltip } = tooltipContent;
 
+	const shortenedNumberConfig: Intl.NumberFormatOptions = {
```

Comment:
I just thought of something 🤔 

This config is being used for stats, videopress, and now here. I am assuming the vast majority of the time we use this function, this will be the config. What do you think about moving this to the utils function as the default config? And we can still have the config as an optional prop. Not necessary to do here but it should be pretty easy 🤔

Reply:
If you don't feel like we should do that in this PR, we can just create a small maintenance task. Just feel like it will be prioritized as "nice to have" 😅

Reply:
@CodeyGuyDylan, yes good idea, nice improvement, I like it! 😃. Thanks for pointing this out! 👍
Ok, I added the default options to the utils/format-number function in this PR.

Could you re-review? Your previous approval was dismissed with the latest changes...

I'll then whip up a quick follow-up PR to remove the shortenedNumberConfig (the second parameter) where the function is used within the Stats card and the VideoPress card. Sounds good? 🙂